### PR TITLE
Adds IndieHosters SaaS offer

### PR DESCRIPTION
**Pull request by pierreozoux**
I'm wondering, should it be in a different category than PaaS offer?

[Review on Reviewable](https://reviewable.io/reviews/wekan/wekan/512)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,5 +40,7 @@ with [Meteor](https://www.meteor.com).
 [docker_image]: https://hub.docker.com/r/mquandalle/wekan/
 [heroku_button]: https://www.herokucdn.com/deploy/button.png
 [heroku_deploy]: https://heroku.com/deploy?template=https://github.com/wekan/wekan/tree/master
+[indiehosters_button]: https://indie.host/signup.png
```
Comment (mquandalle) [inline README.md]: There is no https://indie.host/signup.png

Comment (pierreozoux) [inline README.md]: Yes there is, but I don't understand why Github doesn't want it :/ maybe the extension `.host` is not recognized, I'll ask Github support.

Comment (pierreozoux) [inline README.md]: Arf, sorry you were right... the dns is still propagating... I'll try again tomorrow, and ping you

Comment (pierreozoux) [inline README.md]: Ok, now corrected.

I guess, you'd prefer with the same size?

Comment (mquandalle) [inline README.md]: Yes

Comment (pierreozoux) [inline README.md]: ok updated

**Pull request merged**